Geometry/TrackerGeometryBuilder/interface/phase1PixelTopology.h

```diff
@@ -3,6 +3,7 @@
 
 #include <cstdint>
 #include <array>
+// #include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
```

Comment:
is this needed?

Reply:
not anymore, we prefer NOT to have dependency on HeterogeneousCore in Geometry for the time being